examples/ffdmd/Si-drive/README

```diff
@@ -0,0 +1,24 @@
+This is an example for a time dependent drive in the smotion class involving a solid.
+
+Order of manipulations to get things done:
+  1. Make sure you have i-PI and LAMMPS installed,
+     put proper executable names to run-*.sh
+
+  2. Calculate phonons with i-PI (or with your code of choice)
+
+  3. Convert i-PI normal modes to xyz_jmol format using 'eigvec-to-xyz_jmol.py'
+     (call it without parameters to see the options and ask for script if you don't have it)
```

Comment:
what does this mean? shouldn't the script be included in the PR if it's needed to run the example?

Reply:
Also not relevant anymore.